.github/workflows/assurance-prerequisites.yml

```
@@ -0,0 +1,167 @@
+    uses: ./.github/workflows/portable-assurance.yml
+    with:
+      ref: ${{ inputs.ref || github.ref }}
+      assurance: true
+      assurance_tag: ${{ inputs.assurance_tag }}
+
+  # Fuzzing is advisory: a finding is recorded in the report and does not stop a
+  # release, so a failure here must not fail this workflow either. The verdict
+  # job below is what decides the stage.
+  fuzz:
+    name: Parser fuzzing
+    continue-on-error: true
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "fuzz"

(syntax-check)